zabartcc/api
# Implement Automatic Reminder Emails

## Implement Automatic Reminder Emails

---

### Summary

- Adds logic to send controllers automatic email reminders to control at 45 days, and about removal at 60 days.

closes #22 

---

### Testing

1. Deploy PR to beta environment, with production check turned off, set CRON to every minute, set removal notice to 15 minutes in the future, set to emails to testing emails.
2. Check the admin dashboard and verify the inactive controllers correspond to the emails sent.
3. In 15 minutes verify removal emails were sent.
4. Verify in the DB removal date is nulled, next check date is 60 days in the future

---

### Post-Deployment Validation

1. Check with DATM to verify emails are being sent appropriately and that logs do not show email errors.

---
